go/common/version/version.go

```diff
@@ -50,6 +50,9 @@ func (v Version) MajorMinor() Version {
 }
 
 var (
+	// VersionBuild stores current git HEAD and is set by the linker.
+	VersionBuild = ""
```

Comment:
Since this is used as `version.VersionBuild` it would be better if it was just `Build` so used as `version.Build`. Probably the linter will also complain.